pkg/kubectl/describe/versioned/describe.go

```diff
@@ -3074,20 +3075,31 @@ type HorizontalPodAutoscalerDescriber struct {
 }
 
 func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string, describerSettings describe.DescriberSettings) (string, error) {
-	hpa, err := d.client.AutoscalingV2beta2().HorizontalPodAutoscalers(namespace).Get(name, metav1.GetOptions{})
-	if err != nil {
-		return "", err
+	var events *corev1.EventList
```

Comment:
nits: hold the explicit declaration into `:=`

Reply:
I think this might be more friendly to a code reader.